invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsSearch/components/DocumentSearchResultsGrid/index.js

```diff
@@ -0,0 +1,3 @@
+export {
```

Comment:
Do we need to have its own folder for each component even if they are this simple?

Reply:
I split them because it is easier to responsive views in this way, I'm just using smaller pieces, specialised in one thing, even if it is only different display. Also in this way I'm decreasing the amount of `renderCard` type of functions inside of a component, which in my opinion was making the code a bit messy.